core/model/src/main/kotlin/org/meshtastic/core/model/util/DateTimeUtils.kt

```diff
@@ -62,5 +64,26 @@ private fun formatUptime(seconds: Long): String {
         .joinToString(" ")
 }
 
-@Suppress("MagicNumber")
-fun onlineTimeThreshold() = (System.currentTimeMillis() / 1000 - 2 * 60 * 60).toInt()
+fun onlineTimeThreshold(): Int {
+    val currentSeconds = System.currentTimeMillis() / TimeUnit.SECONDS.toMillis(1)
+    return (currentSeconds - ONLINE_WINDOW_HOURS * TimeUnit.HOURS.toSeconds(ONLINE_WINDOW_HOURS.toLong())).toInt()
```

Comment:
Should this be just `(currentSeconds - TimeUnit.HOURS.toSeconds(ONLINE_WINDOW_HOURS.toLong()))`?

Reply:
Certainly looks like it should be.